packages/node_modules/pouchdb-find/src/adapters/local/abstract-mapper.js

```diff
@@ -23,7 +23,7 @@ function createDeepMultiMapper(fields, emit) {
         var key = parsedField[j];
         value = value[key];
         if (!value) {
```

Comment:
Instead of `if (!value)`, could you do a check if the value is undefined or null. I can't remember if we have an exists:false test. If we don't could you add one in so we can prove it doesn't break exists: false.